src/components/BugzillaComponents/index.jsx

```diff
@@ -105,12 +106,28 @@ const getTableHeaders = (data, onComponentDetails) => {
   };
 
   const getColor = (value, key) => (key === 'P1Defect' && (value && value.count) > 0 ? 'red' : 'blue');
+  // returns columns to be displayed for selection when clicked on view columns
+  const getColumnBool = (dataObject, label, labelString) => {
+    let returningBool = false;
+    const keys = Object.keys(dataObject);
+    keys.forEach((key) => {
+      if (dataObject[key][labelString] === label) {
+        if (key === 'P1Defect' || key === 'unassignedBetaBugs' || key === 'P1Task' || key === 'P1Enhancement' || key === 'sec') {
```

Comment:
I am not a fan of hard coding this list.
What about adding a value in reporteesMetrics https://github.com/mozilla/bugzilla-dashboard/blob/master/src/config.js#L29
hiddenByDefault=true
or something like that?